tests/test_custom_objects_rest.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test that all fields can be replicated for a stream that is a custom object (REST API)
+"""
+from datetime import datetime, timedelta
+
+from tap_tester import connections, menagerie, runner
+
+from base import SalesforceBaseTest
```

Comment:
is this the new base or the old one.  We should be writing all tests against the new one.

Reply:
Will update to new base file.